doc/releases/release-notes-4.0.rst

```diff
@@ -107,6 +107,8 @@ Bluetooth
 
   * Added API :c:func:`bt_gatt_get_uatt_mtu` to get current Unenhanced ATT MTU of a given
     connection (experimental).
+  * Added :kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ`.
+    The option allows using a separate workqueue for connection TX notify processing (:c:func:`bt_conn_tx_notify`) to make Bluetooth stack more independent from the system workqueue.
```

Comment:
Do we have max line length requirements for .rst files?

Reply:
I see that generally doc lines tend to be split if they are longer than 100 characters (but it's not always a case). I will align my note.

Reply:
@kartben Thoughts?

Reply:
Ya in theory it's 100. In practice a better rule would probably be one sentence per line, as it also makes it easier for folks to realize that a blank line is required for paragraph breaks -- when using line wrapping people tend to assume that having just one line return will do that (as it looks "ok enough" in source form) when obviously it doesn't. It also makes diffs cleaner too, since adding a word or two to a line really just impacts that line as opposed to potentially many if it "pushes" a wrapped line to the next (and so on).
Unfortunately switching to this approach would require updating (and potentially breaking) so many files that I have yet to give it a try :)

In the meantime, 100 characters per line it is, when possible :)